tests/test_message.py

```diff
@@ -256,6 +256,13 @@ class Color(proto.Enum):
     s_dict = Squid.to_dict(s)
     assert s_dict["chromatophores"][0]["color"] == 1
 
+    s_dict = Squid.to_dict(
+        s,
+        including_default_value_fields=True,
+        always_print_fields_with_no_presence=True,
+    )
+    assert s_dict["chromatophores"][0]["color"] == 1
```

Comment:
What is this testing? I think we want to check that a non-optional field with the default value is emitted in the dictionary. So we should check assert that  a color value of `UNKNOWN` shows up under these conditions

And when the parameter is false, we should assert that a color value of `UNKNOWN` does NOT show.

I suggest for both of these cases, we modify the class to have two scalar fields, one of which we don't set (or set to the default), and test the four combinations that way. Essentially what we have in the test for `to_json`

Reply:
I fixed the test case in https://github.com/googleapis/proto-plus-python/pull/433/commits/7af40e3357583448ca3167b97a09668daa9c9e35. I wasn't able to get the value of `UNKNOWN` to show up as the default value of the repeated field `chromatophores` is `[]`